src/NServiceBus.Persistence.CosmosDB/Config/CosmosDbPersistenceConfig.cs

```diff
@@ -48,12 +50,22 @@ public static PersistenceExtensions<CosmosDbPersistence> DefaultContainer(this P
         /// <summary>
         /// Disables the container creation.
         /// </summary>
-        public static void DisableContainerCreation(this PersistenceExtensions<CosmosDbPersistence> configuration)
+        public static void DisableContainerCreation(this PersistenceExtensions<CosmosDbPersistence> persistenceExtensions)
         {
-            Guard.AgainstNull(nameof(configuration), configuration);
+            Guard.AgainstNull(nameof(persistenceExtensions), persistenceExtensions);
 
-            var installerSettings = configuration.GetSettings().GetOrCreate<InstallerSettings>();
+            var installerSettings = persistenceExtensions.GetSettings().GetOrCreate<InstallerSettings>();
             installerSettings.Disabled = true;
         }
+
+        /// <summary>
+        /// Enable support for sagas migrated from Azure Storage Persistence.
```

Comment:
I think we should make this more generic. Theoretically we don't care where the data is coming from and thus option enables query on get